cmd/launcher/launcher.go

```diff
@@ -168,13 +170,49 @@ func runLauncher(ctx context.Context, cancel func(), opts *launcher.Options) err
 	flagController := flags.NewFlagController(logger, stores[storage.AgentFlagsStore], fcOpts...)
 	k := knapsack.New(stores, flagController, db)
 
+	slogLevel := slog.LevelInfo
+	if k.Debug() {
+		slogLevel = slog.LevelDebug
+	}
+
+	slogAttrReplacementFunc := func(groups []string, a slog.Attr) slog.Attr {
+		if a.Key != slog.TimeKey {
+			return a
+		}
+
+		return slog.Attr{
+			Key:   slog.TimeKey,
+			Value: slog.AnyValue(time.Now().UTC()),
+		}
+	}
+
+	k.AddLogHandler(slog.NewJSONHandler(os.Stderr, &slog.HandlerOptions{
+		AddSource:   true,
+		Level:       slogLevel,
+		ReplaceAttr: slogAttrReplacementFunc,
+	}))
+
+	k.AddLogHandler(slog.NewJSONHandler(logFileWriter, &slog.HandlerOptions{
```

Comment:
I _think_ but I'm not sure, this might be cleaner if the `locallogger` took a `*slog. Level`, and returned a handler. Though maybe too many of the options should be configured up here?

And maybe we should be standardizing on io.Writers instead of log.Loggers?

Reply:
> And maybe we should be standardizing on io.Writers instead of log.Loggers?
 Maybe? I think it's just weird because we have two now, I would think we would gradually migrate toward slog.

I added a `multislogger` stuct that tracks the handlers and now passing that around in the ctx from main.go -> launcher.go. I think it's cleaner, still had to expose the writer on locallogger, but passing it around.